src/app/pages/send/choose-crypto-asset/components/crypto-asset-list-item.tsx

```diff
@@ -25,6 +31,7 @@ export function CryptoAssetListItem(props: CryptoAssetListItemProps) {
       return (
         <CryptoCurrencyAssetItem
           assetBalance={assetBalance}
+          data-testid={dataTestId}
```

Comment:
In these instances, I'd see no harm declaring the data id directly in the child component. They have the info to construct the same id, and is one less prop to pass.